community.pathofbuilding.PathOfBuilding.desktop

```diff
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=Path of Building (Community)
 Comment=Offline build planer for Path of Exile
-Exec=pob.sh %U
+Exec=pathofbuilding
```

Comment:
Keep %U . The Arch package had the scheme-handler and everything. Though I've never seen it used..